src/gui/rss/rsswidget.cpp

```diff
@@ -455,13 +455,17 @@ void RSSWidget::handleCurrentArticleItemChanged(QListWidgetItem *currentItem, QL
     auto article = m_articleListWidget->getRSSArticle(currentItem);
     Q_ASSERT(article);
 
+    const QColor highlightedBase = m_ui->textBrowser->palette().color(QPalette::Highlight);
+    const QColor highlightedBaseText = m_ui->textBrowser->palette().color(QPalette::HighlightedText);
+    const QColor alternateBase = m_ui->textBrowser->palette().color(QPalette::AlternateBase);
+
     QString html =
-        "<div style='border: 2px solid red; margin-left: 5px; margin-right: 5px; margin-bottom: 5px;'>"
-        "<div style='background-color: #678db2; font-weight: bold; color: #fff;'>" + article->title() + "</div>";
+        "<div style='border: 2px solid red; margin-left: 5px; margin-right: 5px; margin-bottom: 5px;'>" +
+        QString("<div style='background-color: \"%1\"; font-weight: bold; color: \"%2\";'>").arg(highlightedBase.name(), highlightedBaseText.name()) + article->title() + "</div>";
```

Comment:
IMO it would be better to make use of `QString::fromLatin1()`, such as:
```c++
QString html =
    QString::fromLatin1("<div style='border: 2px solid red; margin-left: 5px; margin-right: 5px; margin-bottom: 5px;'>")
    + QString::fromLatin1("<div style='background-color: \"%1\"; font-weight: bold; color: \"%2\";'>%3</div>").arg(highlightedBase.name(), highlightedBaseText.name(), article->title());
```

Reply:
I'll change it but I don't see the point of using QString::fromLatin1 the string will be converted to QString preferred format at runtime anyway

Reply:
It is the same point of using `QLatin1String` isn't?

Reply:
>the string will be converted to QString preferred format at runtime anyway

Conversation from latin-1 string literal should be faster than from utf-8 (even if it really contains latin-1 characters only).